packages/scaffold-ui/src/partials/w3m-all-wallets-list/index.ts

```diff
@@ -53,30 +53,31 @@ export class W3mAllWalletsList extends LitElement {
   public override render() {
     return html`
       <wui-grid
-        data-scroll=${!this.initial}
+        data-scroll=${!this.loading}
         .padding=${['0', 's', 's', 's'] as const}
         columnGap="xxs"
         rowGap="l"
         justifyContent="space-between"
       >
-        ${this.initial ? this.shimmerTemplate(16) : this.walletsTemplate()}
+        ${this.loading ? this.shimmerTemplate(16) : this.walletsTemplate()}
         ${this.paginationLoaderTemplate()}
       </wui-grid>
     `
   }
 
   // Private Methods ------------------------------------- //
   private async initialFetch() {
+    this.loading = true
     const gridEl = this.shadowRoot?.querySelector('wui-grid')
-    if (this.initial && gridEl) {
+    if (gridEl) {
```

Comment:
> Fixes issue where only wallets from the initially active chain ID would be fetched

From the changelog it's hard to understand if this is the fix really - just double checking, does not setting loading to true on the top of the function causing this?

Reply:
Bc the ApiController tests has changed but no changes on the controllers etc

Reply:
test spec changed just cause I abstracted a constant, no real functionality change. 

This change in particular is cause the fetch was previuously done on the first ever render, which was not re-triggered later on. Since I now trigger it on network change as well, it's no longer `initial` but rather `loading`